### ISSUE TITLE
about acdc_dataset

### ISSUE THREAD

**Issue by Yss83** [Beckschen/TransUNet#117]
I'm interested in your project. Could you share the preprocessed acdc_dataset with me please?
If you can't  share the preprocessed acdc_dataset, can you tell me which patients you use as the training set and which patients do the test set
Thank you in advance.

**Comment by Gi-gigi** [comment]
Hi, friends. Did u get the preprocessed acdc_dataset?  Could you share it?

**Comment by Yss83** [comment]
sorry,I did not get acdc_data



------------------&nbsp;原始邮件&nbsp;------------------
发件人: ***@***.***&gt;; 
发送时间: 2023年5月13日(星期六) 下午3:39
收件人: ***@***.***&gt;; 
抄送: ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [Beckschen/TransUNet] about acdc_dataset (Issue #117)





 
Hi, friends. Did u get the preprocessed acdc_dataset?  Could you share it?
